### ISSUE TITLE
"unused result" warning with 0.4.18

### ISSUE THREAD

**Issue by alecmocatta** [taiki-e/pin-project#238]
I just updated from 0.4.17 to 0.4.18 and I now receive the following warning:

```rust
#![warn(unused_results)]

use pin_project::pin_project;

#[pin_project]
pub struct Wrapper<T> {
    #[pin]
    t: T,
}
```

```text
warning: unused result
  --> src/lib.rs:83:3
   |
83 |           #[pin]
   |  _________^
84 | |         t: T,
   | |____________^

warning: unused result
  --> src/lib.rs:83:3
   |
83 |           #[pin]
   |  _________^
84 | |         t: T,
   | |____________^
```

rustc: `nightly-2020-06-04`

**Comment by taiki-e** [comment]
Published 0.4.19 which fixes this issue, thanks @alecmocatta for the reporting!

**Comment by alecmocatta** [comment]
Thanks for the swift fix @taiki-e!
